### ISSUE TITLE
jsonschema~=2.6 dependency causes DeprecationWarning under Python 3.7

### ISSUE THREAD

**Issue by jmehnle** [aws-cloudformation/cfn-python-lint#905]
*cfn-lint version: 0.20.2*

The `jsonschema~=2.6` dependency causes a `DeprecationWarning` under Python 3.7:

```
  …/lib/python3.7/site-packages/jsonschema/compat.py:6: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import MutableMapping, Sequence  # noqa
```

This was [fixed in `jsonschema` 3.0.0](https://github.com/Julian/jsonschema/pull/465). Can you please bump your `jsonschema` dependency to allow (or require) 3.0? ([See also](https://github.com/Julian/jsonschema/issues/487).)

**Comment by kddejong** [comment]
We should be able to update after https://github.com/awslabs/serverless-application-model/issues/922 is complete.

**Comment by kddejong** [comment]
This should be fixed in v0.22.0.
